contributors.md

```diff
@@ -274,28 +292,33 @@ The following people are not part of the development team, but have been contrib
 * Swedish - (Jinxit), (mharrys), (Slimeyo), Matte Andersson (Nubbie)
 
 ## Graphics
-* OpenRCT2 Logo - Yannic Geurts (xzbobzx)
-* Booster Sprites - Nicolas Hawrysh (xp4xbox)
+* Edward Calver (X7123M3-256) - Hybrid RC track & train, Single Rail RC track & train, Alpine RC track & vehicle, LIM RC extended trackset
+* Karst van Galen Last (AuraSpecs) - Steep log flume, Inverted RC Booster
+* Michael Steenbeek (Gymnasiast) - Extended fontset
+* Nicolas Hawrysh (xp4xbox) - Booster Sprites Twister, Giga, Mini, Woodie
+* Yannic Geurts (xzbobzx) - OpenRCT2 Logo
```

Comment:
Once #19919 is merged we also need to add all the graphics artists for this added.